client/my-sites/stats/activity-log/index.jsx

```diff
@@ -363,6 +364,7 @@ class ActivityLog extends Component {
 		const requestedRestoreId = this.props.requestedRestoreId || rewindId;
 		return (
 			<div key={ `end-banner-${ restoreId || downloadId }` }>
+				<PageViewTracker path="/stats/activity/:site" title="Stats > Activity" />
```

Comment:
Oh this needs to be moved under main right? This component file is pretty huge 😅

Reply:
Nice catch! :smile: Yeah, I somehow inserted this here instead in the main render method. :man_facepalming:

Reply:
Updated in cf356b76e0b54ab3f34140730ca6f42e9afaeff9.